src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -346,20 +347,27 @@ class ACCLoopDirective(ACCRegionDirective):
                              to the loop directive.
     :param bool sequential: whether or not to add the `seq` clause to the \
                             loop directive.
+    :param bool gang: whether or not to add the `gang` clause to the \
```

Comment:
I only learnt this relatively recently but we don't need the line-continuation (`\`) symbols so long as we indent the following lines (which we do).

Reply:
Ah okay, thanks. Done in 1357686886bb0627676797bd1b251ed91508be5a.